_data/store/helsinkisystems-lexoffice.yaml

```diff
@@ -0,0 +1,11 @@
+developer: helsinkisystems
+icon: fas fa-file-invoice
+demo: false
+subscription: 99 €
+lemonsqueezy: "https://helsinkisystems.lemonsqueezy.com/checkout?cart=eef22eb9-992a-4d66-97a9-9abc4b04dc08"
+bundle:
+    name: "Lexoffice"
```

Comment:
The namespace Lexoffice is pretty generic. If it is only about Invoices, then please rename it to LexofficeInvoice